modules/attribution-document/attribution-document-generator/src/test/java/org/eclipse/sw360/antenna/attribution/document/workflow/generators/YagenIntegrationTest.java

```diff
@@ -31,25 +34,52 @@
  * in the tempdir.
  */
 class YagenIntegrationTest {
+   private static final String  productName = "My Awesome Product with a very long name, " +
+           "that also contains brackets and stuff";
+   private static final String version = "abc123";
+   private static final String copyrightHolder = "My Legal Entity GmbH";
 
    @TempDir
    File tmpDir;
 
    @Test
    void doGenerate() {
-      String productName = "My Awesome Product with a very long name, that also contains brackets and stuff";
-      String version = "abc123";
-      String copyrightHolder = "My Legal Entity GmbH";
+      DocumentValues values = createDocumentValue(productName, version, copyrightHolder);
 
-      DocumentValues values = new DocumentValues(productName, version, copyrightHolder);
-
-      AttributionDocumentGeneratorImpl attrDocGen = new AttributionDocumentGeneratorImpl("attribution_document.pdf", tmpDir, "antenna-demo", values);
+      AttributionDocumentGeneratorImpl attrDocGen =
+              new AttributionDocumentGeneratorImpl("attribution_document.pdf", tmpDir, "antenna-demo", values);
 
       List<ArtifactAndLicense> foo = createData();
       File file = attrDocGen.generate(foo);
       assertThat(file).exists().hasExtension("pdf");
    }
 
+   @Test
+   void goGenerateWithFiles() throws IOException {
+      int expectedResultSize = 189490;
```

Comment:
Maybe again inline this (as "pdf" below is also not put into a variable, and it makes more clear at a single glance what the expected file size is)?